lit_llama/__init__.py

```diff
@@ -1,2 +1,2 @@
-from lit_llama.model import LLaMAConfig, LLaMA, RMSNorm, build_rope_cache, apply_rope
+from lit_llama.model import LLaMAConfig, LLaMA, RMSNorm, llama_model_lookup, build_rope_cache, apply_rope
```

Comment:
I would not expose `llama_model_lookup` here. I would just import it from `lit_llama.utils` where needed.

Reply:
Seems reasonable, done!